dataduct/s3/utils.py

```diff
@@ -119,6 +125,40 @@ def upload_dir_to_s3(s3_path, local_path, filter_function=None):
         local_path(file_path): Input path of the file to be uploaded
         filter_function(function): Function to filter out directories
     """
+
+    def multipart_upload(key_string, local_file_path):
+        username = os.path.basename(os.path.expanduser('~'))
+        directory = '/tmp/multipart_upload_{}'.format(username)
+        if not os.path.exists(directory):
+            os.makedirs(directory)
+        prefix = os.path.join(directory, 'tmp_upload')
+        # split file into parts
+        split = \
```

Comment:
try not to use continuation line style when possible, this can look something like
```
["split",
  "-b..."]
```